### PR TITLE
fix: benches compilation and all tests

### PR DESCRIPTION
**Pull request by joske**

### CODE REVIEW

File: narwhal/primary/src/aggregators.rs [hunk 1]
```diff
@@ -68,9 +69,7 @@ impl VotesAggregator {
                     while i < self.votes.len() {
                         let (id, sig) = &self.votes[i];
                         let pk = committee.authority_safe(id).protocol_key();
-                        if sig
-                            .verify_secure(&to_intent_message(certificate_digest), pk)
-                        {
+                        if !sig.verify_secure(&to_intent_message(certificate_digest), pk) {
                             warn!("Invalid signature on header from authority: {}", id);
                             self.weight -= committee.stake(pk);
                             self.votes.remove(i);
```
Comment (SplittyDev) [inline narwhal/primary/src/aggregators.rs:75]: Great catch 👍

**Review by SplittyDev — APPROVED** [review]
Works perfectly!

**Review by ljedrz — APPROVED** [review]
```
Summary [ 119.329s] 271 tests run: 271 passed (1 slow), 12 skipped
```

LGTM :+1:

**Review by niklaslong — APPROVED** [review]
Ditto

**Pull request merged**